aci/login/login.go

```diff
@@ -38,12 +38,18 @@ import (
 
 //go login process, derived from code sample provided by MS at https://github.com/devigned/go-az-cli-stuff
 const (
-	authorizeFormat = "https://login.microsoftonline.com/organizations/oauth2/v2.0/authorize?response_type=code&client_id=%s&redirect_uri=%s&state=%s&prompt=select_account&response_mode=query&scope=%s"
-	tokenEndpoint   = "https://login.microsoftonline.com/%s/oauth2/v2.0/token"
-	getTenantURL    = "https://management.azure.com/tenants?api-version=2019-11-01"
+	// AcrRegistrySuffix suffix for ACR registry images
+	AcrRegistrySuffix         = ".azurecr.io"
+	activeDirectoryURL        = "https://login.microsoftonline.com"
+	azureManagementURL        = "https://management.core.windows.net/"
+	azureResouceManagementURL = "https://management.azure.com/"
```

Comment:
```suggestion
	azureManagementURL        = "https://management.core.windows.net"
	azureResouceManagementURL = "https://management.azure.com"
```
What about normalizing the end of the constant so it's responsability of the user to specify the "/"?
This makes explicit from the reader that the previous section of the URL is completed and whatever is concatenated is another section.

Reply:
Actually I tried to stick as much as possible with values we get if you try to do `az cloud list` so that if we want to switch clouds we can easily replace these const.

Reply:
OK